RecoLocalCalo/HGCalRecProducers/python/hgcalLayerClusters_cff.py

```diff
@@ -26,10 +29,11 @@
     nHitsTime = cms.uint32(3),
```

Comment:
```suggestion
    nHitsTime = 3,
```
or can be dropped because it's the same in the original `hgcalLayerClusters_`